neutronics/mcnp_inputs.py

```diff
@@ -31,20 +31,22 @@ class HomogeneousInput:
     """Write Homogeneous Input File.
     Class to write homogeneous MCNP burnup input files.
     """
-    
-    def __init__(self, radius, length, pnnl_mats, thick_refl=15):
+    kW_to_MW = 0.001    
+    def __init__(self, radius, length, power, pnnl_mats, thick_refl=15):
         """Initialize geometric reactor parameters.
 
         Initialized Attributes:
         -----------------------
             z (float): reactor core height
             r (float): reactor core radius
             frac_fuel (float): fuel to coolant channel fraction
+            Q_therm (float): reactor thermal power
             matlib (MaterialLibrary): PyNE material library
         """
         self.z = length
         self.r = radius
         self.refl_t = thick_refl
+        self.Q_therm = power * self.kW_to_MW
```

Comment:
Why user have to provide power in MW? why not the same as the stored one ?
Why storing the value in kW ? Why not W ?